### DISCUSSION TITLE
p_skey问题

### DISCUSSION THREAD

**Discussion by oppsDayly** [sun589/QQkey_Tool#23] [提问]
我使用您的脚本获取到p_skey后放到qq频道的请求里面为什么显示是无效的？

**Comment by sun589** [comment]
请先描述详细一些，至少描述一下内容，屑屑 ：） 
1.哪个频道/干什么的
2.获取的哪个p_skey
3.是否确定p_skey没过期（时间过长会过期）

**Reply by oppsDayly** [reply]
没事了谢谢
